### PR TITLE
Implement podman-remote wait command and container subcommand

### PR DESCRIPTION
**Pull request by jwhonce**
Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: cmd/podman/wait.go [hunk 1]
```diff
@@ -49,43 +49,36 @@ func waitCmd(c *cliconfig.WaitValues) error {
 		return errors.Errorf("you must provide at least one container name or id")
 	}
 
-	runtime, err := libpodruntime.GetRuntime(&c.PodmanCommand)
+	if c.Interval == 0 {
```
Comment (TomSweeneyRedHat) [inline cmd/podman/wait.go]: s/==/<=/   just to be ultra safe?

**Comment by TomSweeneyRedHat** [comment]
One small comment for consideration.  Tests aren't happy and it's not entirely clear why not....

**Comment by TomSweeneyRedHat** [comment]
bot, retest this please

File: API.md [hunk 2]
```diff
@@ -1013,10 +1013,10 @@ VolumesPrune removes unused volumes on the host
 ### <a name="WaitContainer"></a>func WaitContainer
 <div style="background-color: #E8E8E8; padding: 15px; margin: 10px; border-radius: 10px;">
 
-method WaitContainer(name: [string](https://godoc.org/builtin#string)) [int](https://godoc.org/builtin#int)</div>
-WaitContainer takes the name or ID of a container and waits until the container stops.  Upon stopping, the return
-code of the container is returned. If the container container cannot be found by ID or name,
-a [ContainerNotFound](#ContainerNotFound) error is returned.
+method WaitContainer(name: [string](https://godoc.org/builtin#string), interval: [int](https://godoc.org/builtin#int)) [int](https://godoc.org/builtin#int)</div>
+WaitContainer takes the name or ID of a container and waits the given interval in milliseconds until the container
```
Comment (mheon) [inline API.md]: Maybe "waits, polling at the given interval (in milliseconds), until the container stops"

File: cmd/podman/wait.go [hunk 3]
```diff
@@ -49,43 +49,36 @@ func waitCmd(c *cliconfig.WaitValues) error {
...
-		}
-		returnCode, err := ctr.WaitWithInterval(time.Duration(c.Interval) * time.Millisecond)
-		if err != nil {
-			if lastError != nil {
-				fmt.Fprintln(os.Stderr, lastError)
-			}
-			lastError = errors.Wrapf(err, "failed to wait for the container %v", container)
-		} else {
-			fmt.Println(returnCode)
+	if len(failures) > 0 {
```
Comment (mheon) [inline cmd/podman/wait.go]: You used very similar code in the stop PR - can that be refactored into a shared helper?

Comment (jwhonce) [reply cmd/podman/wait.go]: Already done and included in PR for rm.

File: API.md (hunk 2 above)
Comment (jwhonce) [reply API.md]: @mheon Do we want to share that implementation detail in an API document?

**Comment by baude** [comment]
yeah LGTM

**Pull request merged**